pkg/apis/proto/daemon/daemon.proto

```diff
@@ -23,6 +23,19 @@ message BufferInfo {
   required bool isFull = 11;
 }
 
+// ProcessingRate is used to store the lookback duration and also the corresponding rate.
+message ProcessingRate {
+  required string lookback = 1;
+  required float rate = 2;
+}
+
+// VertexInfo is used to provide information about the vertex including processing rate.
+message VertexInfo {
```

Comment:
`VertexInfo` is too big for it I think - there's no vertex information provided in it. Maybe use a different name like `VertexAddtionalInfo` (I'm not good at naming, please find one).

Reply:
I named it `VertexMetrics`